retrofit-adapters/rxjava/src/test/java/retrofit2/adapter/rxjava/ObservableTest.java

```diff
@@ -86,83 +81,16 @@
 
     RecordingSubscriber<String> subscriber = subscriberRule.createWithInitialRequest(0);
     service.body().unsafeSubscribe(subscriber);
-    assertThat(server.getRequestCount()).isEqualTo(0);
+    assertThat(server.getRequestCount()).isEqualTo(1);
+    subscriber.assertNoEvents();
```

Comment:
This is an unrelated internal behavior change I've been meaning to make. It has no effect on what the downstream consumer of the stream sees in terms of behavior.